release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,16 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+// TODO: replace "TODO" with appropriate z-stream, and is "check" the correct term?
+{product-title} 4.8.TODO introduced a check that impacts upgrading to the next {product-title} release, which is currently planned to be {product-title} 4.9. This is because Kubernetes 1.22, which {product-title} 4.9 is expected to use, removed a significant number of deprecated `v1beta1` APIs.
```

Comment:
Can give a related hyper-link to `significant number of deprecated v1beta1 APIs`?  Just like 4.9.

Reply:
That list unfortunately only exists in the 4.9 release notes, not here in the 4.8 release notes. But the list is available in the kbase article that is linked in the last line of the section

Reply:
That's not very convenient, we can give one related Kubernetes doc link, I get from 4.9  For more information about migrating removed APIs, see the [Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22).

Reply:
@wangke19 I added that link to the Kubernetes docs here, let me know if that works for you!